nidm/nidm-experiment/terms/nidm-experiment.owl

```diff
@@ -10,6 +10,7 @@
 @prefix xsp: <http://www.owl-ontologies.com/2005/08/07/xsp.owl#> .
 @prefix daml: <http://www.daml.org/2001/03/daml+oil#> .
 @prefix meta: <http://purl.obolibrary.org/obo/iao/dev/ontology-metadata.owl#> .
+@prefix nidm: <http://purl.org/nidash/nidm/experiment#> .
```

Comment:
@nicholsn: yes, I agree with this update. But as this is currently under discussion at #237, I would suggest that we merge this PR first and update the namespace when we close #237.